controllers/archived.js

```diff
@@ -23,22 +27,16 @@ legacyPagePaths.forEach(urlPath => {
 
 /**
  * Archived files
- * Requests for legacy files (eg. stored on Sitecore) will be shown an error message
+ * Requests for legacy files will be shown an error message
  * along with a feedback form to explain what they were looking for.
- * We also log all requests for these files to ensure we can update anything missing.
  */
 router.get(legacyFilesPath, (req, res) => {
-    const filePath = req.originalUrl;
-    metrics.count({
```

Comment:
I'm removing this because this is also getting tracked as a page-view in analytics which is more readable. The cloudwatch metrics aren't very readable in this case.